docs/core/diagnostics/sos-debugging-extension.md

```diff
@@ -0,0 +1,369 @@
+|    dumpdomain                       | Displays information all the AppDomains and all assemblies within the domains.
+|    dumpheap                         | Displays info about the garbage-collected heap and collection statistics about objects.
+|    dumpil                           | Displays the Microsoft intermediate language (MSIL) that is associated with a managed method.
+|    dumplog                          | Writes the contents of an in-memory stress log to the specified file.
+|    dumpmd                           | Displays information about a MethodDesc structure at the specified address.
+|    dumpmodule                       | Displays information about a EE module structure at the specified address.
+|    dumpmt                           | Displays information about a method table at the specified address.
+|    dumpobj                          | Displays info about an object at the specified address.
+|    dumpstack                        | Displays a native and managed stack trace.
+|    eeheap                           | Displays info about process memory consumed by internal runtime data structures.
+|    eestack                          | Runs dumpstack on all threads in the process.
+|    gcroot                           | Displays info about references (or roots) to an object at the specified address.
+|    histclear                        | Releases any resources used by the family of Hist commands.
+|    histinit                         | Initializes the SOS structures from the stress log saved in the debuggee.
+|    histobj                          | Examines all stress log relocation records and displays the chain of garbage collection relocations that may have led to the address passed in as an argument.
+|    histobjfind                      | Displays all the log entries that reference an object at the specified address.
+|    histroot                         | Displays information related to both promotions and relocations of the specified root.
+|    ip2md                            | Displays the MethodDesc structure at the specified address in code that has been JIT-compiled.
+|    loadsymbols                      | Load the .NET Core native module symbols.
+|    name2ee                          | Displays the MethodTable structure and EEClass structure for the specified type or method in the specified module.
+|    pe                               | Displays and formats fields of any object derived from the Exception class at the specified address.
+|    setclrpath                       | Set the path to load coreclr dac/dbi files. setclrpath `<path`>
+|    sethostruntime                   | Sets or displays the .NET Core runtime directory to use to run managed code in SOS.
+|    setsymbolserver                  | Enables the symbol server support.
+|    setsostid                        | Set the current os tid/thread index instead of using the one lldb provides. setsostid `<tid`> `<index`>
```

Comment:
```suggestion
|    setsostid                        | Set the current OS tid/thread index instead of using the one lldb provides: `setsostid <tid> <index>`
```